doc/comparison.md

```diff
@@ -0,0 +1,25 @@
+# Comparison
+
+| Tool    | Binary | Source Code/LLVM | Available Under-constrained Algorithm |
+|---------|--------|------------------|---------------------------------------|
+| [angr](https://angr.io/)    | Yes    | No*              | Yes*                                   |
+| [Grease](https://github.com/GaloisInc/grease)  | Yes    | Yes              | Yes                                   |
+| [KLEE](https://github.com/klee/klee)  | No     | Yes              | No                                    |
+| UC-KLEE | No     | Yes              | Yes                                   |
+| [UC-Crux](https://www.galois.com/articles/under-constrained-symbolic-execution-with-crucible) | No     | Yes              | Yes                                   |
+| [Macaw](https://github.com/GaloisInc/macaw)   | Yes    | No               | No                                    |
```

Comment:
Macaw's inclusion in this table feels a bit strange, as it's a library to build other things on, not a standalone tool. Moreover, GREASE is built on top of Macaw. I feel as though it might be better to leave out Macaw, as it's not an apples-to-apples comparison with things like GREASE.

Reply:
I kinda agree but on the other hand angr sits in the same place imo where we are talking about basic building blocks with forward symb ex and code recovery

Reply:
Hm, OK. I think I suggested the title "Comparison with other tools" under the assumption that we would only compare it to standalone tools like GREASE, not libraries like Macaw. But fair enough, if we were to leave out Macaw, then you could just as well argue that we should leave out Angr, which wouldn't feel right...

In that case, perhaps we should retitle this section to something like "Comparison with other tools and frameworks"?

Reply:
yeah i think that works for me and i hope my blurb about frameworks like macaw/angr as building blocks for directly usable tools helps